skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
```

Comment:
@vighneshbirodkar I will be called a lot of times during the training.
Rough example: if we use `20x20` sized training examples, on each image we will
have to call this function `2049` times on each training image.
So if we have `20000` training images, it will be `20000x2049` for each epoch of training.